### PR TITLE
Transfer the `colored` designation into Option type

### PR DESCRIPTION
**Pull request by sahara-ooga**
I propose that the coloring specification be included as part of the Option type.
IMHO,  this will make it possible to specify the coloring in a way that is appropriate to the frequency of use, thereby eliminating the bother.

# Motivation

The option to color the output is not likely to be changed frequently during development. 
Therefore, the current format of specifying it every time in a method argument such as `prettyPrint` is likely to create a hassle.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -286,6 +286,20 @@ Pretty.prettyPrint(value, option: Pretty.Option(prefix: nil, indentSize: 2))
...
+
+```swift
+Pretty.sharedOption = Pretty.Option(colored: true)
+let value = (bool: true, array: ["Hello", "World"])
+Pretty.prettyPrint(value)
+```
+
+This output looks like as bellow:
+
+<img src=https://raw.githubusercontent.com/YusukeHosonuma/SwiftPrettyPrint/main/Image/colored-option.png>
```
Comment (YusukeHosonuma) [inline README.md]: Please use relative-path to reference image.

File: README.md [hunk 2]
```diff
@@ -286,6 +286,20 @@ Pretty.prettyPrint(value, option: Pretty.Option(prefix: nil, indentSize: 2))
 // )
 ```
 
+### colorized
+
+Output strings can be colored.
```
Comment (YusukeHosonuma) [inline README.md]: Please write more detail description.

For example:

- Colorlized are works by ANSI color.
- Therefore it works only terminal that supported ANSI color e.g. AppCode, Terminal.app (This is ** not include Xcode**).
- Related to [terminal](https://github.com/YusukeHosonuma/SwiftPrettyPrint#terminal) section.

**Review by YusukeHosonuma — CHANGES_REQUESTED** [review]
I think the code is fine 👍 
However, I feel the documentation is lacking, so please add an explanation 🙏

File: README.md (hunk 1 above)
Comment (sahara-ooga) [reply README.md]: Fine. Fixed.

8230ed69264b7c5b2daf97f2c089773be101ffb7

On the other hand, some of the existing descriptions in REAEME.md are written in absolute paths as in the pointed out sections.

I've created a separate PR #174 to eliminate that confusion, so please review that as well.
@YusukeHosonuma

File: README.md (hunk 2 above)
Comment (sahara-ooga) [reply README.md]: I've added detail descriptions.
https://github.com/YusukeHosonuma/SwiftPrettyPrint/pull/173/commits/f60f6cbe5f21d44cfb968f875b262289753d8acb

File: README.md [hunk 3]
```diff
@@ -286,6 +286,24 @@ Pretty.prettyPrint(value, option: Pretty.Option(prefix: nil, indentSize: 2))
...
+```swift
+Pretty.sharedOption = Pretty.Option(colored: true)
+let value = (bool: true, array: ["Hello", "World"])
+Pretty.prettyPrint(value)
+```
+
+This output looks like as bellow:
+
+<img src=Image/colored-option.png>
+
```
Comment (YusukeHosonuma) [inline README.md]: I improved the text a little (and fix-typo `bellow` -> `below`).

````suggestion
```swift
Pretty.sharedOption = Pretty.Option(colored: true)
let value = (bool: true, array: ["Hello", "World"])
Pretty.prettyPrint(value)
```

This output looks like as below:

<img src=Image/colored-option.png>

It works only on console that ANSI color supported (e.g. AppCode, Terminal.app).
This does **not** includes Xcode debug console.

See also [Terminal](#Terminal) section.

````

How about it?

**Review by YusukeHosonuma — COMMENTED** [review]
I improved the text a little.
Make a final check 🙏

**Review by YusukeHosonuma — APPROVED** [review]

File: README.md (hunk 3 above)
Comment (sahara-ooga) [reply README.md]: I appreciate your suggestion!
I marged it👍🏻

**Pull request merged**